src/phoenix/server/api/types/Project.py

```diff
@@ -226,76 +226,66 @@ def document_evaluation_names(
         )
 
     @strawberry.field
-    def trace_evaluation_summary(
+    async def trace_evaluation_summary(
         self,
+        info: Info[Context, None],
         evaluation_name: str,
         time_range: Optional[TimeRange] = UNSET,
     ) -> Optional[EvaluationSummary]:
-        project = self.project
-        eval_trace_ids = project.get_trace_evaluation_trace_ids(evaluation_name)
-        if not eval_trace_ids:
-            return None
-        trace_ids = project.get_trace_ids(
-            start_time=time_range.start if time_range else None,
-            stop_time=time_range.end if time_range else None,
-            trace_ids=eval_trace_ids,
+        stmt = (
+            select(models.TraceAnnotation)
+            .join(models.Trace)
+            .where(models.Trace.project_rowid == self.id_attr)
+            .where(models.SpanAnnotation.annotator_kind == "LLM")
+            .where(models.SpanAnnotation.name == evaluation_name)
```

Comment:
```suggestion
            .where(models.TraceAnnotation.annotator_kind == "LLM")
            .where(models.TraceAnnotation.name == evaluation_name)
```

Reply:
Suggestion: I am not sure if it makes a difference in terms of performance, but I would probably use `and_` here since it maps more closely to how I would write this in SQL.